CloudyCider/views/components/EC2/EC2List.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  EC2List.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import SwiftUI
+
+struct EC2List: View {
+  @ObjectBinding var store: EC2Store
+  @State var pageListener = EC2PageListener()
+
+  var body: some View {
+    HStack {
+      if !store.errorMessage.isEmpty {
```

Comment:
this should be checked against `nil` with `if let` after `errorMessage` is made optional